java/tv/sage/ModuleGroup.java

```diff
@@ -293,27 +293,32 @@ else if (cryptoKeyObj instanceof Object[])
         }
       }
       file.createNewFile();
-      sage.FastRandomFile widgetDBout = new sage.FasterRandomFile(file, cryptoKey == null ? "rwd" : "rwdc", sage.Sage.I18N_CHARSET);
-      if (cryptoKey != null) widgetDBout.setCrypto(cryptoKey);
+      sage.io.SageDataFile widgetDBout;
+
+      if (cryptoKey == null)
```

Comment:
Lol...you can still remove it...but apparently it wasn't just SageTVLite, I think we actually encrypted all the .stv files originally until we switched to XML files. Geeze...I was paranoid. :)